apps/bookmark/js/activity_handler.js

```diff
@@ -10,10 +10,10 @@ var ActivityHandler = {
           window.removeEventListener('status-hidden', hidden);
           activity.postResult(saved ? 'saved' : 'updated');
         });
-        
+
         utils.status.show(
           navigator.mozL10n.get(saved ? 'added-to-home-screen-message' :
```

Comment:
I know is not from this PR, but can we use .formatValue instead? I think we are trying to remove .get